internal/limitopen/limitopen.go

```diff
@@ -9,10 +9,28 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
 	"github.com/reddit/baseplate.go/log"
 	"github.com/reddit/baseplate.go/metricsbp"
 )
 
+const (
+	pathLabel = "path"
+)
+
+var (
+	sizeLabels = []string{
+		pathLabel,
+	}
+
+	sizeGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "limitopen_file_size_bytes",
```

Comment:
Let's use `Namespace` here. It's also good to make this a package const.

```suggestion
const (
	metricsNamespace = "limitopen"
)
		Namespace: metricsNamespace,
		Name: "limitopen_file_size_bytes",
```

Reply:
is namespace similar to a meta label? also do we want to repeat namespace and the name? (here the namespace is also the prefix of the name)

Reply:
how does namespace show up in grafana? we did not use namespaces in any prometheus metrics defined in the spec.